src/audio/Audio.js

```diff
@@ -56,7 +56,11 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		this.buffer = audioBuffer;
 		this.sourceType = 'buffer';
 
-		if ( this.autoplay ) this.play();
```

Comment:
I actually prefer how this line was already. That's one of the issues I have with enforcing the eslint stuff...